src/Data/List/Fresh/NonEmpty.agda

```diff
@@ -0,0 +1,47 @@
+{-# OPTIONS --without-K --safe #-}
```

Comment:
You'll need a header briefly describing the module here (cf. the start of other files).